### PR TITLE
PKs Rebalancing Mod: Fungus

### PR DESCRIPTION
**Pull request by pisskop**
Fungus amoungus

### CODE REVIEW

File: data/mods/PKs_Rebalance/pk_critter_factions.json [hunk 1]
```diff
@@ -12,6 +21,19 @@
...
+        "by_mood" : ["small_animal", "blob", "utility_bot", "mutant_omnivore"]
+    },{
+        "type" : "MONSTER_FACTION",
+        "name" : "mutant_omnivore",
+        "neutral" : ["utility_bot", "small_animal"],
+        "by_mood" : ["blob", "mutant_predator", "mutant"]
+    },{
+        "type" : "MONSTER_FACTION",
+        "name" : "mutant_predator",
+        "by_mood" : ["mutant_omnivore", "blob"]
```
Comment (Coolthulhu) [inline data/mods/PKs_Rebalance/pk_critter_factions.json]: You shouldn't explicitly declare factions hostile when they're already hostile.

**Comment by Coolthulhu** [comment]
Json errors on load.

**Comment by pisskop** [comment]
Mutant factions arent inherently hostile.  I plan to eventually expand the group until it largely outcrowds the standard wildlife.  Not all will be hostile to one another.

**Comment by Coolthulhu** [comment]
> Mutant factions arent inherently hostile.

Doesn't matter, they're still hostile towards the blob faction by default. No need to specify that thrice for each faction.

**Comment by pisskop** [comment]
Its but flavoring so sure

**Comment by Coolthulhu** [comment]
Json errors on load

File: data/mods/PKs_Rebalance/pk_itemlist.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      ["spear_wood", 5],
+      ["spear_forked", 5],
+      ["spear_steel", 5],
+      ["spear_copper", 3],
+      ["sword_crude", 3],
+      ["homewrecker", 3],
+      ["halberd_fake", 3],
+      ["javelin_iron", 3],
+      ["bwirebat", 3],
+	  ["machete", 3],
```
Comment (Coolthulhu) [inline data/mods/PKs_Rebalance/pk_itemlist.json]: Tabs

**Comment by Coolthulhu** [comment]
I think it conflicts with the doom part of the mod. Will need the conflicts resolved.

**Comment by pisskop** [comment]
Right.

So you cant have 2 PRs up that edit 1 file then, because they cant resolve together.

I did want to test that.  Ill just make this later

**Pull request closed without merging**